workspaces/ui-v2/src/pages/docs/DocumentationRootPage/DocumentationRootPage.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { FC, useMemo } from 'react';
```

Comment:
This is mainly the same page - just moved over (git cant tell because there's a fair amount of changes in this file)